interface/src/components/swap/virtualized-tokens-list.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import { VariableSizeList as List } from 'react-window'
+import AutoSizer from '@brave/react-virtualized-auto-sizer'
+
+// Types
+import { BlockchainToken } from '~/constants/types'
+
+// Components
+import { TokenListButton } from '~/components/buttons'
+import Amount from '~/utils/amount'
+
+interface VirtualizedTokensListProps {
+  tokenList: BlockchainToken[]
+  onSelectToken: (token: BlockchainToken) => void
+  getAssetBalance: (token: BlockchainToken) => Amount
+  disabledToken: BlockchainToken | undefined
+  isWalletConnected: boolean
+}
+
+interface ListItemProps extends Omit<VirtualizedTokensListProps, 'tokenList'> {
+  index: number
+  data: BlockchainToken[]
+  style: React.CSSProperties
+}
+
+const itemSize = 72
```

Comment:
What is this value based on?

Also, are the contents of this file similar to what we do in brave-core?

Reply:
The height of each item in the list:

![Screen Shot 2022-10-26 at 8 07 10 AM](https://user-images.githubusercontent.com/40611140/198033789-02d4f819-a93a-4c9d-bdf0-183775f60d84.png)

Yes they are very similar.